test/graphemetest.c

```diff
@@ -49,7 +51,7 @@ int main(int argc, char **argv)
                 else
                     i++;
             }
-            glen = utf8proc_map(utf8, j, &g, UTF8PROC_CHARBOUND);
+            glen = utf8proc_map(utf8, j, &g, UTF8PROC_CHARBOUND | UTF8PROC_OLDENCODE);
```

Comment:
I'd missed that (about CHARBOUND...).
OK, I'll try to get some testing in... (in my copious spare time! :grinning:)
Would it be possible to have extra testing as a follow-up PR?